packages/backend-core/src/utils/utils.ts

```diff
@@ -214,6 +214,36 @@ export async function getBuildersCount() {
   return builders.length
 }
 
+/**
+ * Logs a user out from budibase. Re-used across account portal and builder.
+ */
+export async function platformLogout(opts: PlatformLogoutOpts) {
```

Comment:
This has been moved to the `auth` package now. Must have been re-added in a merge.